### PR TITLE
Hotfix/#2044 fix

### PR DESCRIPTION
**Pull request by beatrizsmerino**
# Related issue
[BUG] Nuxt static page is shifted #2044

# Scope of work

Wrong styles in 'sf-header-navigation-item' links
- Hidden the left element of the link that is empty
- Prevent link text from being broken into words and left in columns
- Added missing semicolon

Wrong styles of the cart button:
- Get the size of the button and calculate its half to move it below.
- Use the css variable ‘—button-box-shadow’ alrready declarated for the css property 'box-shadow'

# Screenshots of visual changes
image
image

# Checklist

- [x] No commented blocks of code left
- [x] Run tests and docs
- [x] Self code-reviewed
- [x] Changes documented 

If applicable:

- [x] I followed [composition rules](https://docs.storefrontui.io/?path=/story/introduction-contributing-guide-code-guidelines--page) for my component
- [x] I tested the component in most common device sizes (can be tested in Storybook [from viewport addon in top menu](https://github.com/storybooks/storybook/tree/master/addons/viewport))

### CODE REVIEW

**Comment by AdamPawlinski** [comment]
@beatrizsmerino Hi, thanks for the great work, it looks fine, but  could you please take a look on `ProductCard` component with color picker? When you check it on storyook docs it looks like this 
Screenshot from 2021-11-02 12-05-39

The button is too low. Do you think you could handle that bug?

**Comment by beatrizsmerino** [comment]
Of course, I will take a look at it and try to fix it.

**Comment by aniamusial** [comment]
@beatrizsmerino how its going? do you think you can finish this today?

**Comment by beatrizsmerino** [comment]
Hi @aniamusial
I've been busy, but I'll get to it this week!

**Comment by aniamusial** [comment]
@beatrizsmerino hey, do you need any help with finishing this task?

**Review by AdamPawlinski — APPROVED** [review]

File: packages/shared/styles/components/organisms/SfHeader.scss [hunk 1]
```diff
@@ -15,7 +15,8 @@
     --header-navigation-item-padding: 0;
   }
   &__link {
-    --link-text-decoration: var(--header-navigation-item-text-decoration, none)
+    --link-text-decoration: var(--header-navigation-item-text-decoration, none);
+    display: none;
```
Comment (AdamPawlinski) [inline packages/shared/styles/components/organisms/SfHeader.scss:19]: I can't see navigation items with display: none.
Screenshot from 2021-11-23 08-20-44

File: packages/shared/styles/components/organisms/SfProductCard.scss [hunk 2]
```diff
@@ -142,6 +143,9 @@
   &.on-wishlist {
     --product-card-wishlist-icon-opacity: 1;
   }
+  & .has-colors {
```
Comment (aniamusial) [inline packages/shared/styles/components/organisms/SfProductCard.scss:146]: chore: remove the spacing

**Review by aniamusial — APPROVED** [review]

**Review by AdamPawlinski — APPROVED** [review]

**Comment by beatrizsmerino** [comment]
> @beatrizsmerino hey, do you need any help with finishing this task?

In the end I didn't have time to do it, I'm sorry.

**Comment by justyna-13** [comment]
No worries @beatrizsmerino :) if you don't have much time but still want to contribute then you can take an issue that doesn't have assigned a version - for example these: https://github.com/vuestorefront/storefront-ui/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22

**Pull request merged**